app/javascript/scripts/forms.js

```diff
@@ -0,0 +1,16 @@
+export default () => {
+  const userCurrentPassword = document.querySelector('#user_current_password');
+  const updateUserSubmit = document.querySelector('#update-user-submit');
+
+  if (userCurrentPassword && updateUserSubmit) {
+    userCurrentPassword.addEventListener('input', (event) => {
+      if (event.target.value) {
```

Comment:
You can simplify this condition.

If you've done this b/c of known upcoming changes, it's good practice to leave a comment, ideally as a `TODO`, with a note of the expected behaviour change/addition.